pkg/cmd/metadata.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/puppetlabs/relay/pkg/cluster"
+	"github.com/puppetlabs/relay/pkg/debug"
+	"github.com/puppetlabs/relay/pkg/dev"
+	"github.com/puppetlabs/relay/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newMetadataCommand() *cobra.Command {
+
+	// TODO Add help about usage, idling and direct execution
+	cmd := &cobra.Command{
+		Use:   "metadata",
+		Short: "Run a mock metadata service for entrypoint testing",
+		FParseErrWhitelist: cobra.FParseErrWhitelist{
+			UnknownFlags: true,
+		},
+		RunE:               doRunMetadata,
+	}
+
+	cmd.Flags().StringP("input", "i", "", "Path to metadata mock file")
+	_ = cmd.MarkFlagRequired("input")
```

Comment:
Not that we should suppress errors regularly (I agree with it here), but if you want to suppress errors from commands that return one value, you can just call the function without assignment: `cmd.cmd.MarkFlagRequired("input")`

Reply:
Ah, my linter already suggests doing `_ = ...` to explicitly ignore unhandled errors and I usually ignore the suggestion, but let it do its thing here. I guess disabling a linter by name is more clear than adding garbage code to satisfy the linter. So I guess that's acually even a bad practice introduced by GoLand.